contributors/guide/github-workflow.md

```diff
@@ -133,7 +133,7 @@ echo export PATH="\$PATH:$working_dir/kubernetes/third_party/etcd" >> ~/.profile
 ```sh
 cd $working_dir/kubernetes
 
-# Run all the presubmission verification. Then, run a specific update script (hack/update-*.sh)
+# Run all the pre-submission verification. Then, run a specific update script (hack/update-*.sh)
```

Comment:
`presubmission` is correct: http://www.yourdictionary.com/presubmission.

Reply:
Thank you. I've just reverted it.